src/Sulu/Bundle/AdminBundle/Resources/js/containers/CKEditor5/ckeditor5.scss

```diff
@@ -25,6 +25,9 @@ $textEditorUnpublishedLinkColor: $gold;
         --ck-font-size-base: $textEditorFontSize;
         --ck-icon-size: $textEditorFontSize;
 
+        /* z-index */
+        --ck-z-default: 0;
```

Comment:
what happens if we set this to `auto` here instead of `0` does that work?

Reply:
That doesn't work unfortunately 😬

Reply:
also when setting `--ck-z-modal` to auto?:

so:

```scss
--ck-z-default: auto;
--ck-z-modal: auto;
```

Reply:
what is the behaviour with auto?

Reply:
```css
/* z-index */
--ck-z-default: auto;
--ck-z-modal: auto;
--ck-balloon-panel-arrow-z-index: auto;
```

If I do so, it's the same behaviour as before

![image](https://user-images.githubusercontent.com/5758674/104312007-473f5700-54d6-11eb-91fb-ac515a2da5b6.png)

Reply:
Since this solution is working and I didn't find an edge case, where it doesn't work, I don't think it's necessary to further improve this. But if you want to investigate more, feel free to take over this pull request 😉

Reply:
okay for me. can you test also the following case @nnatter posted here https://github.com/sulu/sulu/pull/5138#issuecomment-604490692

Reply:
looks good

Reply:
good from my side. think would be nice if @nnatter could test it has he knows most about the edge cases with the zindexes.

Reply:
yes, would also appreciate that 🙂